CSS modifications

Rukmini-Meda opened the pull request:
<!-- Add the issue number that is fixed by this PR (Eg: Fixes #45) -->
Fixes #4 

#### Changes done in this pull request:
<!-- Add a small decription to your solution -->
 Changed styling of the input fields

#### Files Added:
<!-- Mention any new files added,removed or changed-->
No new files added or removed.
index.js and App.css files changed




#### Screenshots/Links:
<!-- Add revelant screenshots/links to view your work-->

![Screenshot (139)](https://user-images.githubusercontent.com/44504940/80784961-5d9dc080-8b9c-11ea-8f25-320c9200176a.png)

![Screenshot (140)](https://user-images.githubusercontent.com/44504940/80784972-67bfbf00-8b9c-11ea-819e-39469aca22ae.png)

![Screenshot (141)](https://user-images.githubusercontent.com/44504940/80784975-6b534600-8b9c-11ea-8ce5-73b7fad93d44.png)



#### System Information :
<!-- Add the relevant information about the system used during development-->
 - OS: Windows
 - Browser: Google Chrome
 - Version: 81.0.4044.129 (Official Build) (64-bit)



#### Reviewers:
<!-- Tag in maintainers to review your work-->
@AdityaSrivast Please review my PR. I made style changes for input fields and error dialog. Please let me know if I have to make any more further changes or improvements.


> We appreciate you help towards improving the module, we look forward to having more contributions from your end

Rukmini-Meda commented:
@AdityaSrivast Please review my PR. Please let me know if I have to improve anything.

AdityaSrivast commented on test-server/package.json:
```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
+    "react-pincode": "^1.0.2",
```
@Rukmini-Meda there's no need to install the module in the test repo. Get rid of this line.

AdityaSrivast commented on src/App.css:
```diff
@@ -10,10 +15,43 @@ input::-webkit-inner-spin-button {
 /* Firefox */
 input[type=number] {
-  -moz-appearance: textfield;
+  /* -moz-appearance: textfield; */
```
@Rukmini-Meda any reason why this line is commented?

AdityaSrivast replied:
@Rukmini-Meda ?

Rukmini-Meda commented:
@AdityaSrivast I made changes as you said. I removed the package-lock.json file, removed the line you said in package.json. And about the line that's commented, when I started to implement my styling I kinda commented it to check what I was doing. Its completely fine even if its not commented. Please review for any more changes and merge. Thank you!

Rukmini-Meda commented:
@AdityaSrivast Can you merge this PR if there are no other changes or improvements required?

AdityaSrivast commented:
@Rukmini-Meda get rid of `package-lock.json`. Also if possible give any codesandbox, or jsfiddle link so that your implementation can be verified

Rukmini-Meda commented:
@AdityaSrivast Sure, I will update it by tomorrow.

Rukmini-Meda commented:
@AdityaSrivast Can you help me with codesandbox? I imported my branch into it. But build/index.js shows many problems although it works fine in vscode. This is the link for codesandbox - https://codesandbox.io/s/distracted-yonath-6l27d?file=/build/index.js:0-26767

Rukmini-Meda commented:
> @AdityaSrivast Can you help me with codesandbox? I imported my branch into it. But build/index.js shows many problems although it works fine in vscode. This is the link for codesandbox - https://codesandbox.io/s/distracted-yonath-6l27d?file=/build/index.js:0-26767

@AdityaSrivast Can you help with this so that the PR can be merged as we have only 2 days left?

Rukmini-Meda commented:
@yellowwoods12 Can you suggest any resources?

Rukmini-Meda commented:
@yellowwoods12 @AdityaSrivast Can you please help me with codesandbox so that I can complete this PR successfully?

AdityaSrivast commented:
@Rukmini-Meda use `test-server` github url while importing, and add the code for module like `Pincode.js` and paste your code from `App.js` to it. Also add your css file. That should do the trick.
